lib/WebpackConfig.js

```diff
@@ -99,6 +99,22 @@ class WebpackConfig {
         this.eslintLoaderOptionsCallback = () => {};
         this.tsConfigurationCallback = () => {};
         this.handlebarsConfigurationCallback = () => {};
+        this.loaderConfigurationCallbacks = {
+            javascript: () => {},
+            js: () => {},
```

Comment:
Not sure we should have two callbacks for `javascript` and `js` there, an alias directly handled by `applyRuleConfigurationCallback` would probably avoid weird cases.

Same remark for `sass`/`scss` and `typescript`/`ts`

Reply:
That's should be better now.